go/v1/statement.go

```diff
@@ -0,0 +1,40 @@
+/*
+Wrapper APIs for in-toto attestation Statement layer protos.
+*/
+
+package v1
+
+const StatementTypeUri = "https://in-toto.io/Statement/v1"
+
+func (s *Statement) Validate() bool {
+	if s.GetType() != StatementTypeUri {
+		return false
```

Comment:
As with ResourceDescriptor perhaps this could return an error instead of bool?

Reply:
done.